pyradiosky/tests/test_skymodel.py

```diff
@@ -1389,7 +1389,7 @@ def test_select(spec_type, time_location):
     skyobj = SkyModel.from_gleam_catalog(GLEAM_vot)
 
     skyobj.beam_amp = np.ones((4, skyobj.Nfreqs, skyobj.Ncomponents))
-    skyobj.extended_model_group = np.arange(skyobj.Ncomponents)
+    skyobj.extended_model_group = np.empty(skyobj.Ncomponents, dtype=str)
```

Comment:
Can we add a test to ensure that the complicated logic around making source IDs unique and setting the component IDs are doing what we want them to? Maybe by making a tiny little dummy catalog that we can then assert has exactly the expected names and extended_model_group names?